library/src/main/java/uk/co/samuelwall/materialtaptargetprompt/extras/backgrounds/CirclePromptBackground.java

```diff
@@ -174,12 +174,11 @@ else if (x3 > focalLeft && x3 < focalRight)
         else
         {
             mBasePosition.set(focalCentreX, focalCentreY);
-            // Calculate the furthest distance from the a focal side to a text side.
-            final float length = Math.abs(
-                    (textBounds.left < focalBounds.left ?
-                            textBounds.left - textPadding
-                            : textBounds.right + textPadding)
-                    - focalCentreX);
+           // Calculate the furthest distance from the center based on the text size.
+            final float length = Math.max(
+                    Math.abs(textBounds.right - focalBounds.centerX()),
+                    Math.abs(textBounds.left - focalBounds.centerX())
+            ) + textPadding + focalBounds.width() / 2f
```

Comment:
Missing a `;` from the end of the line which results in a compilation error

Reply:
My fault there, my whole code is in Kotlin less this class. My mistake when I made the PR